features/polypolyMembershipPreview/src/locales/en/preview.json

```diff
@@ -1,6 +1,6 @@
 {
-    "progress.banner.title": "We have an idea",
-    "progress.banner.description": "Brainstorming leads us here. We are seriously thinking about this, but did not start working on it yet. ",
+    "progress.banner.title": "We sketch",
```

Comment:
Shouldn't we put all these common strings in poly-look? Or some other place that's a single source of truth.

Reply:
I would love that!
There is already a single source of truth in the [progress.info](https://github.com/polypoly-eu/polyPod/blob/d265bc8d43bd63d75aa76af5385525e909647669/features/polypolyMembershipPreview/src/locales/en/progressInfo.json#L1) file 

I would have liked to connect these to the content.json `progress_banner.stage`
IF there is nothing in the locales overriding the default text. Because in the future we might need to override the default text.

I talked about this with @timoteipalade , I don't know if there is a ticket yet for it

Reply:
Search for one and/or create one if there's not. I don't think there is one.